src/api/tests/fixtures/collect_records.py

```diff
@@ -240,3 +239,218 @@ def invalid_collect_record_error(
         stage=CollectRecord.VALIDATED_STAGE,
         data=data_error,
     )
+
+
+@pytest.fixture
+def valid_benthic_lit_collect_record(
+    benthic_attribute_3,
+    benthic_attribute_4,
+    project1,
+    profile1,
+    project_profile1,
+    sample_event1,
+):
+    observations = [
+        dict(attribute=str(benthic_attribute_3.id), length=1000),
+        dict(attribute=str(benthic_attribute_3.id), length=1500),
+        dict(attribute=str(benthic_attribute_3.id), length=2000),
+        dict(attribute=str(benthic_attribute_3.id), length=2500),
+        dict(attribute=str(benthic_attribute_4.id), length=3000),
+    ]
+    data_ok = dict(
+        protocol="benthiclit",
+        obs_benthic_lits=observations,
+        benthic_transect=dict(depth=1, number=2, len_surveyed=100),
+        sample_event=dict(
+            management=str(sample_event1.management.id),
+            site=str(sample_event1.site.id),
+            sample_date=f"{sample_event1.sample_date:%Y-%m-%d}",
+        ),
+        observers=[{"profile": str(profile1.id)}],
+    )
+    return CollectRecord.objects.create(
+        project=project1,
+        profile=profile1,
+        stage=CollectRecord.VALIDATED_STAGE,
+        data=data_ok,
+    )
+
+
+@pytest.fixture
+def invalid_benthic_lit_collect_record(
+    benthic_attribute_3,
+    project1,
+    profile1,
+    project_profile1,
+    sample_event1,
+):
+    observations = [
```

Comment:
Would it be considered bad form to DRY up all the pieces in the valid/invalid versions that are the same, but add the two observations for the valid version? Had to think a bit on why this was invalid.

Reply:
I was on the fence with this and leaned towards readability and keeping it in one place